aiosmtpd/smtp.py

```diff
@@ -570,10 +550,7 @@ def smtp_DATA(self, arg):
             else:
                 status = self.event_handler.process_message(*args)
```

Comment:
The assignment to local variable "status" is not needed here and should be removed.